platform/api/equinixmetal/api.go

```diff
@@ -362,7 +362,8 @@ ExecStart=/usr/bin/curl --retry-delay 1 --retry 120 --retry-connrefused --retry-
 # We don't verify signatures because the iPXE script isn't verified either
 # (and, in fact, is transferred over HTTP)
 
-ExecStartPre=-/bin/bash -c 'shopt -s nullglob; for disk in /dev/*da? /dev/nvme?n1; do wipefs --all --force ${disk}; done'
+ExecStartPre=-/bin/bash -c 'lvchange -an /dev/mapper/*'
+ExecStartPre=-/bin/bash -c 'shopt -s nullglob; for disk in /dev/*da? /dev/nvme?n1; do wipefs --all --force $${disk}; done'
```

Comment:
```suggestion
ExecStartPre=-/bin/bash -c 'shopt -s nullglob; for disk in /dev/*d? /dev/nvme?n1; do wipefs --all --force $${disk}; done'
```

Reply:
This made me realize the previous approach was slightly flawed

Reply:
Related, do you know when there would be a `n2` suffix for the nvme device?

Reply:
Those are nvme namespaces, but I think they are only supported by advanced drives, which we won't come across.

Reply:
Should we use `?d?`?

Reply:
In case this runs in other other environment there could be `/dev/xda`, or?

Reply:
Changed it back to `*d` as prefix to capture `xvda`, too